tests/test_integration.py

```diff
@@ -1055,12 +1059,59 @@ async def test_info_links(links):
     assert stderr == ""
 
 
+@pytest.mark.asyncio
+async def test_info_remote_data(envsonschema):
+    jsonout, stderr = await bowtie(
```

Comment:
This isn't what I had in mind, as it still will hit the internet -- you can see that yourself by trying to run this test with your computer not connected to the internet -- as I say, in order to not do that, you need to be able to inject in the fake response below *without* actually making a call to the internet. I'll try to look more closely and show you how to do so this weekend.